File: csharp/SwimmingCompetitionCS/model/Organizer.cs
using System;

namespace model
{
    [Serializable]
    public class Organizer : Entity<long>
    {
        private string _username;
        private string _password;

        public Organizer(string username, string password)
        {
            this._username = username;
            this._password = password;
        }

        public string Password
        {
            get => _password;
            set => _password = value;
        }

        public string Username
        {
            get => _username;
            set => _username = value;
        }

        public override string ToString()
        {
            return "Organizer{" +
                   "id=" + Id +
                   ", username='" + _username + '\'' +
                   ", password='" + _password + '\'' +
                   '}';
        }
    }
}

File: java/SwimmingCompetition/SwimmingCompetitionModel/src/main/java/swimmingcompetition/model/Organizer.java
package swimmingcompetition.model;


public class Organizer extends Entity<Long>{
    private String username;
    private String password;

    public Organizer(String username, String password) {
        this.username = username;
        this.password = password;
    }

    public String getUsername() {
        return username;
    }

    public void setUsername(String username) {
        this.username = username;
    }

    public String getPassword() {
        return password;
    }

    public void setPassword(String password) {
        this.password = password;
    }

    @Override
    public String toString() {
        return "Organizer{" +
                "id=" + id +
                ", username='" + username + '\'' +
                ", password='" + password + '\'' +
                '}';
    }
}
